subprojects/base-services/base-services.gradle.kts

```diff
@@ -26,19 +26,16 @@ dependencies {
     implementation(library("commons_io"))
     implementation(library("asm"))
 
-    jmhImplementation(library("bouncycastle_provider")) {
-        version {
-            prefer(libraryVersion("bouncycastle_provider"))
-        }
-    }
-
     integTestImplementation(project(":logging"))
-    
+
     testFixturesImplementation(library("guava"))
     testImplementation(testFixtures(project(":core")))
     testRuntimeOnly(library("xerces"))
-    
+
     integTestRuntimeOnly(project(":runtimeApiInfo"))
+
+    jmh("org.bouncycastle:bcprov-jdk15on:1.61")
```

Comment:
What's the reason for `jmhImplementation` not working here?
And neither is `library("bouncycastle_provider")`

Reply:
Can you try with rc2 and see if it fixes the pb?

Reply:
Pushed, it's working the same way with rc1 and rc2 -> only with `jmh`, not with `jmhImplementation` (getting `java.lang.NoClassDefFoundError: org/bouncycastle/jcajce/provider/digest/MD5$Digest`)